src/main/java/racingcar/controller/GameController.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.controller;
+
+import racingcar.IOConsole;
+import racingcar.model.Car;
+import racingcar.service.GameService;
+
+import java.util.List;
+
+public class GameController {
+    private GameService gameService;
+
+    public void play(){
+        List<Car> cars = IOConsole.readCarNames().stream().map(Car::new).toList();
+        int trial = IOConsole.readTrial();
+
+        gameService = new GameService(cars, trial);
```

Comment:
```
Class GameServce {
  private param1
  private param2
  private GameService(param1, param2){}

  public static GameService CreateGameService (param1, param2) {
     // can add try pattern or nullable type
     return new GameService(param1, param2)
  }
}
...

// when call the create static method instead of constructor
GameService.CreateGameService(param1, param2)
```

Domain 에서 constructor보다 static method를 쓰면 
nullable과 try pattern을 사용할 수 있습니다.

추가적으로 Input으로 들어오는 데이터를 IOConsole에서 받아서 처리하기보다는
GameService 도메인 서비스로서 validator를 추가하여, 기획의 의도를 도메인 객체의 비지니스로직으로 남기면 더 좋을듯 합니다.